src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionKind.cs

```diff
@@ -43,5 +43,10 @@ internal enum ConversionKind : byte
         IntPtr,
         InterpolatedString, // a conversion from an interpolated string to IFormattable or FormattableString
         Deconstruction, // The Deconstruction conversion is not part of the language, it is an implementation detail 
+
+        // IdentityValue is not a part of the language. 
+        // It is used by lowering to ensure that trivially reduced expressions 
+        // do not become exposed to mutations if used as receivers of struct methods.
+        IdentityValue,  
```

Comment:
Ok, will keep IdentityValue then

---
In reply to: [108232708](https://github.com/dotnet/roslyn/pull/18166#discussion_r108232708) [](ancestors = 108232708)